stripe/src/main/java/com/stripe/android/paymentsheet/ui/PrimaryButton.kt

```diff
@@ -17,6 +18,7 @@ internal class PrimaryButton @JvmOverloads constructor(
     attrs: AttributeSet? = null,
     defStyleAttr: Int = 0
 ) : FrameLayout(context, attrs, defStyleAttr) {
+    private var defaultTintList: ColorStateList? = null
```

Comment:
Updated this to remove the tint when animating and showing success.  We still want the green background.